problem_builder/templates/html/mentoring-table-report.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ title }}</title>
+    <style>
+    body {
+        font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    }
+    {{css|safe}}
+    </style>
+</head>
+<body>
+<div class="mentoring">
+    <div class="identification">
+        {% if student_name %}{% trans "Student" %}: {{student_name}}<br>{% endif %}
+        {% if course_name %}{% trans "Course" %}: {{course_name}}<br>{% endif %}
+        {% trans "Date" %}: {% now "DATE_FORMAT" %}<br>
+    </div>
+    REPORT_GOES_HERE
```

Comment:
This string is replaced by [review_blocks.js](https://github.com/open-craft/problem-builder/blob/432c1b9a7a42b0d405de83acacee6b1ad4879b69/problem_builder/public/js/review_blocks.js#L54) with the table that's already been rendered to the page, and then delivered as an encoded dataURI so that students can download the report.

I agree it isn't the prettiest mechanism, but the string itself never gets seen.